### ISSUE TITLE
Clog website

### ISSUE THREAD

**Issue by PascalPrecht** [thoughtram/clog#52]
Hey everyone,

I'm currently working on a nice little website for the project so we can promote it a little bit better and also because, awesome projects deserve awesome websites. It's really just about making a nice landing page with all the needed information to get started and some links to the GitHub repo and the docs (which we will publish on gh-pages through this repo).

Here's a screenshot of the work-in-progress:

![screen shot 2015-07-08 at 16 53 19](https://cloud.githubusercontent.com/assets/445106/8581798/4e9ecef6-25c3-11e5-950d-c21ffd498946.png)

Would love to hear your thoughts! Feedback is highly welcome. Also, it'd be nice if we could brainstorm a little bit about the content of the site. Maybe you have some cool ideas.

In addition to that, @Nevraeka is so kind to help us out and works on a very nice logo too, based on this scribble:
![screen shot 2015-07-08 at 22 50 12](https://cloud.githubusercontent.com/assets/445106/8581873/bae9a324-25c3-11e5-8ef3-44145def3f8d.png)

**Comment by PascalPrecht** [comment]
Here's what it looks like on mobile btw:
![screen shot 2015-07-08 at 22 57 35](https://cloud.githubusercontent.com/assets/445106/8582072/d58d90ae-25c4-11e5-8430-4d0634f8e99e.png)
![screen shot 2015-07-08 at 22 57 47](https://cloud.githubusercontent.com/assets/445106/8582073/d91cc5e6-25c4-11e5-89c5-3af91df18603.png)

**Comment by kbknapp** [comment]
I think it looks awesome! :+1: 

I'd be really cool to see some changelog images (not just links to changelogs) in the design as well, or just as examples. I'm not a design guy though, so take my ideas with a grain of salt! :P

**Comment by PascalPrecht** [comment]
@kbknapp Yes, I totally agree. We should somehow get some changelog examples in there so people can actually see what they get.

**Comment by cburgdorf** [comment]
Just wanted to let you know that I put together a "use clog directly from the browser"-demo site. It's build with nickel.rs and uses clog as a library. It's not shiny and pretty at this point but we can iterate on it to have it nicely integrated into the planned clog project site.

I'm going to start writing a blog post soon that leverages the work that I've put into this. Once that's done I'll be pushing the code forward to be useful for a proper project site. E.g. let the user configure the various options of clog through the interface.

I put a preview up here: https://nickel-test.herokuapp.com/
The code is here: https://github.com/thoughtram/clog.thoughtram.io

@kbknapp I know your Rust is much better than mine, if have some time to spare to look through the code, give feedback or send PRs to improve it, I'm always grateful to learn from you.

**Comment by kbknapp** [comment]
@cburgdorf That's super cool! I've just looked briefly though the repo and it looks good! Something that's really impressive to me is how short and concise the code is to do something like this using `clog` and `nickel.rs`! I'm looking forward to this blog and final product!

I'll keep an eye on the issues section of that repo and help out where I can of course! :+1: 

Also, I think incorporating something like this into the `clog` web page would be perfect! I'd put up a very short somewhat curated list of repositories known to use conventional commit messages though (perhaps just 3 or so), since running it on a repo without those would be of limited use :P And it would also add a few different changelogs to see versus the default angular one.

**Comment by PascalPrecht** [comment]
Love your collaborations here :heart:

**Comment by PascalPrecht** [comment]
![screen shot 2015-07-13 at 12 06 40](https://cloud.githubusercontent.com/assets/445106/8647367/aee8b58c-2957-11e5-8712-c3373ef0b6c3.png)

@Nevraeka Hey Erik I might be wrong, but it seems like the logo is not entirely centered. Also, do you think we could squeeze it a bit vertically? Would love to hear your thoughts on that!

**Comment by PascalPrecht** [comment]
Here's a desktop version with the logo embedded:

![screen shot 2015-07-13 at 12 14 34](https://cloud.githubusercontent.com/assets/445106/8647504/c72cc3bc-2958-11e5-9704-93c1bd3af1fe.png)

**Comment by PascalPrecht** [comment]
Here's a little update:

![screen shot 2015-07-13 at 17 22 30](https://cloud.githubusercontent.com/assets/445106/8653339/279898cc-2984-11e5-9782-e1023abb42fd.png)
![screen shot 2015-07-13 at 17 22 46](https://cloud.githubusercontent.com/assets/445106/8653341/2c519d64-2984-11e5-92ec-847e71231395.png)

**Comment by PascalPrecht** [comment]
Current state can be viewed here: https://github.com/PascalPrecht/clog-website

**Comment by kbknapp** [comment]
:heart_eyes:

**Comment by PascalPrecht** [comment]
Okay, I screwed it up. Lost the whole repository while upgrading my machine (because I didn't make a proper back up of it). So I'll have to rebuild the site again  😒

**Comment by PascalPrecht** [comment]
The new repository for the website is here: https://github.com/clog-tool/clog-tool.github.io

Pushing to `master` should serve the files of this repo statically at clog-tool.github.io
